Apsitvarkom.UnitTests/Models/Validation/PollutedLocationDTOValidationTests.cs

```diff
@@ -0,0 +1,106 @@
+﻿using Apsitvarkom.Models.DTO;
+using FluentValidation;
+
+namespace Apsitvarkom.UnitTests.Models.Validation;
+
+public class PollutedLocationDTOValidationTests
+{
+    private static readonly IValidator<PollutedLocationDTO> _validator = new PollutedLocationDTOValidator();
```

Comment:
```suggestion
    private static readonly IValidator<PollutedLocationDTO> Validator = new PollutedLocationDTOValidator();
```

Consider using PascalCase for static variable

Reply:
ahh yeah my bad, this is basically a `const` for reference types, PascalCase preferred.